src/app/user-profile/user-profile.component.ts

```diff
@@ -33,4 +33,22 @@ export class UserProfileComponent implements OnInit {
     .subscribe(data => this.currentUser = data);
   }
 
+  deleteUser(id: number) {
+    if(confirm(`Are you sure you want to delete this user?  This cannot be undone.`)) {
```

Comment:
Can we get the pop up box to ask "Are you sure you want to delete your account? This cannot be undone."